### PR TITLE
Fix static errors

### PR DESCRIPTION
**Pull request by FlXlT**
Fixed a part of the Satic Errors for now.

### CODE REVIEW

**Comment by FlXlT** [comment]
Fixing conflicts.

File: app/src/main/java/nl/tudelft/pixelperfect/client/ClientListener.java [hunk 1]
```diff
@@ -11,13 +11,17 @@
 import nl.tudelft.pixelperfect.event.PlasmaLeakEvent;
 
 /**
+<<<<<<< HEAD
+ * The ClientListeners waits for incoming messages from the server and interpret them.
+=======
  * The ClientListener waits for incoming messages from the server and interps them.
+>>>>>>> release
```
Comment (dmalarev) [inline app/src/main/java/nl/tudelft/pixelperfect/client/ClientListener.java]: I think you missed a conflict here.

**Comment by dmalarev** [comment]
The rest looks good

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/MainActivity.java [hunk 2]
```diff
@@ -28,7 +28,7 @@
      * Whenever this activity is created, the game will be initialized and a dialog will lead
      * the user to the next page.
      *
-     * @param savedInstanceState
+     * @param savedInstanceState , a Bundle.
```
Comment (dalderliesten) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/MainActivity.java:31]: A bit more information is helpful in the future.

**Comment by dalderliesten** [comment]
Looks good to me, watch the checkstyle suppression.

**Pull request merged**